lms/envs/common.py

```diff
@@ -3347,7 +3347,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 ECOMMERCE_SERVICE_WORKER_USERNAME = 'ecommerce_worker'
 ECOMMERCE_API_SIGNING_KEY = 'SET-ME-PLEASE'
 
-COURSE_CATALOG_API_URL = 'http://localhost:8008/api/v1'
+COURSE_CATALOG_URL_ROOT = 'http://localhost:8008'
+COURSE_CATALOG_API_URL = '{}/api/v1'.format(COURSE_CATALOG_URL_ROOT)
```

Comment:
**Question:** What about overriding the complete URL instead of just ROOT URL? What if in future version `v2` comes and we want to use the new version?

Reply:
@saleem-latif On a second look I am wondering the `COURSE_CATALOG_URL_ROOT` will override?

Reply:
Both of these settings will get overridden by https://github.com/edx/edx-platform/blob/master/lms/envs/production.py#L92